truss/truss_config.py

```diff
@@ -78,6 +79,7 @@ class TrussConfig:
     secrets: Dict[str, str] = field(default_factory=dict)
     description: str = None
     bundled_packages_dir: str = DEFAULT_BUNDLED_PACKAGES_DIR
+    spec_version: str = DEFAULT_SPEC_VERSION
```

Comment:
It would be good if this were at the top of the config, especially when it's dumped to yaml.

Reply:
It would also be good to document the format of spec_version. It seems to be neither integer, nor semantic, but in another PR I see it's converted to floating point. Would be good to document the format well.